app/server/utils/user.py

```diff
@@ -231,7 +231,8 @@ def create_transfer_account_user(first_name=None, last_name=None, preferred_lang
             transfer_account.approve_and_disburse()
 
         db.session.add(transfer_account)
-        db.session.commit()
+        #TODO: if i don't have this, default_transfer_account_id is null, if i do, API crashes...
+        #db.session.commit()
 
     user.default_transfer_account_id = transfer_account.id
```

Comment:
was this the bit where i suggested on a different PR doing 
 `user.default_transfer_account = transfer_account`
and u were like `nah doesn't work?`

Reply:
not sure why i turned ur quote into a code block, but you're welcome :P

Reply:
yeah, tried it and still was null

Reply:
might be because we don't have a line similar to `default_organisation`?

```
    default_transfer_account_id = db.Column(db.Integer, db.ForeignKey('transfer_account.id'))

    default_organisation_id = db.Column(
        db.Integer, db.ForeignKey('organisation.id'))

    default_organisation = db.relationship('Organisation',
                                           primaryjoin=Organisation.id == default_organisation_id,
                                           lazy=True,
                                           uselist=False)
```

because if i add it, then c i r c u l a r d e p e n d e n c y